chapter01/Intro_example_short.ipynb

```diff
@@ -0,0 +1,671 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Probably the simplest BIP in the world (the short story)\n",
+    "\n",
+    "Here we define and solve a very simple Bayesian inverse problem (BIP) using \n",
+    "CUQIpy. The purpose of this notebook is to introduce the basic concepts of\n",
+    "Bayesian inverse problems in a simple way and to use minimal CUQIpy code to\n",
+    "solve it.\n",
+    "\n",
+    "In the next notebooks (the long story), we discuss more details about setting up\n",
+    "this BIP in CUQIpy and provide many exercises to help the reader to explore \n",
+    "using CUQIpy in solving BIPs and think of slightly different BIP modeling \n",
+    "scenarios.\n",
+    "\n",
+    "\n",
+    "## <font color=#CD853F> Contents of this notebook: </font>\n",
+    "  * [0. Learning objectives](#r-learning-objectives)\n",
+    "  * [1. Defining the BIP](#r-defining-the-bip)\n",
+    "      * [1.1. The forward model](#r-the-forward-model)\n",
+    "      * [1.2. The prior](#r-the-prior)\n",
+    "      * [1.3. The noise distribution](#r-the-noise-distribution)\n",
+    "      * [1.4. The data distribution](#r-the-data-distribution)\n",
+    "      * [1.5. The likelihood function](#r-the-likelihood-function)\n",
+    "      * [1.6. Putting it all together, the BIP](#r-putting-it-all-together-the-bip)\n",
+    "\n",
+    "  * [2. Solving the BIP](#r-solving-the-bip)\n",
+    "      * [2.2. Maximum a posteriori (MAP) estimate](#r-maximum-a-posteriori-map-estimate)\n",
+    "      * [2.3. Sampling from the posterior](#r-sampling-from-the-posterior)\n",
+    "\n",
+    "  * [3. Summary](#r-summary)\n",
+    "\n",
+    "  * [References](#r-references)\n",
+    "\n",
+    "\n",
+    "\n",
+    "## <font color=#CD853F> 0. Learning objectives </font> <a name=\"r-learning-objectives\"></a>\n",
+    "  * Create a simple BIP in CUQIpy\n",
+    "      * Create a linear forward model object\n",
+    "      * Create distribution objects that represent the prior, the noise, and the data distributions\n",
```

Comment:
data "distribution" instead of "distributions"?

Reply:
the word "distributions" is meant for prior, noise, and data as in prior distribution, noise distribution, and data distribution.